### PR TITLE
Stat tracker refactor

### PR DESCRIPTION
**Pull request by jKwibe**

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,41 +1,29 @@
...
-    end
 
-    StatTracker.new
+  def initialize(location)
+    @games = GameCollection.new(location[:games])
+    @teams = TeamCollection.new(location[:teams])
+    @game_teams = GameTeamCollection.new(location[:game_teams])
+  end
+  def self.from_csv(location)
+    StatTracker.new(location)
```
Comment (Strompy) [inline lib/stat_tracker.rb:15]: Much cleaner now!

**Comment by Truthvendor** [comment]
This new format is a lot more understandable for where the data comes from.

File: lib/stat_tracker.rb (hunk 1 above)
Comment (ggerdsen) [reply lib/stat_tracker.rb:15]: Looks great, these changes will probably help in future iterations.

**Review by Truthvendor — APPROVED** [review]
This new format is a lot more understandable from where the data comes from.

**Pull request merged**